docs/fargate.md

```diff
@@ -471,30 +548,187 @@ spec:
 done | tee -a sumo-metrics-pvc.yaml
 ```
 
-#### Create namespace if it doesn't exist already
+Create namespace if it doesn't exist already
 
 ```bash
 ## Create namespace if it doesn't exist already
 kubectl create namespace "${NAMESPACE}"
 ```
 
-#### Apply `sumo-metrics-pvc.yaml`
+Apply `sumo-metrics-pvc.yaml`
 
 ```bash
 ## Apply `sumo-metrics-pvc.yaml`
 kubectl apply -f sumo-metrics-pvc.yaml -n "${NAMESPACE}"
 ```
 
-### Install or upgrade collection
+#### Create `sumo-logs-pvc.yaml` with PVC per access point
 
-Disable Prometheus Node Exporter. As Amazon Fargate does not support Daemonsets, they won't be working correctly. There is no need to have
-Node Exporter on Amazon Fargate as Amazon takes care of Nodes management. To disable Prometheus Node Exporter, please add the following
-configuration to `user-values.yaml`:
+The objects can be created using the following `bash` script:
```

Comment:
Do you actually need to do all this? Can't you just create the StorageClass and then set `metadata.persistence.storageClass` to the name?

Reply:
yes, those persistence volumes and PVCs need to be created manually, this is the tedious prerequisite of deploying the helm chart with persistence enabled
https://aws.amazon.com/blogs/aws/new-aws-fargate-for-amazon-eks-now-supports-amazon-efs/

Reply:
Ok, dynamic provisioning is not currently supported on Fargate, we can't do any better.